app/assets/javascripts/Components/Modals/submission_url_submit_modal.jsx

```diff
@@ -16,6 +16,10 @@ class SubmitUrlUploadModal extends React.Component {
 
   onSubmit = event => {
     event.preventDefault();
+    this.setState({
```

Comment:
`setState` is asynchronous meaning it _might_ update the state after the `this.props.onSubmit` is triggered but then again it might update it before, and then you'll end up passing empty strings as arguments to that function.